deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,129 @@
+import { CometInterface, CometProxyAdmin, CometRewards, Configurator, ERC20, GovernorSimple, IGovernorBravo, ProxyAdmin, SimpleTimelock } from '../../../../build/types';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { debug } from '../../../../plugins/deployment_manager/Utils';
+import { extractCalldata } from '../../../../src/utils';
+import { exp } from '../../../../test/helpers';
+
+
+interface Vars { };
+
+export default migration('1659582050_raise_supply_caps_and_seed_reserves_and_rewards', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, vars: Vars) => {
+    const { ethers } = deploymentManager.hre;
+
+    // XXX this only has the ABI for the governor proxy
+    const govProxy = await deploymentManager.contract('governor');
+    const governor = await ethers.getContractAt(
+      'IGovernorBravo',
+      govProxy.address,
+    ) as IGovernorBravo;
+    const comet = await deploymentManager.contract('comet') as CometInterface;
+    const configurator = await deploymentManager.contract('configurator') as Configurator;
+    const proxyAdmin = await deploymentManager.contract('cometAdmin') as CometProxyAdmin;
+    const rewards = await deploymentManager.contract('rewards') as CometRewards;
+    const USDC = await deploymentManager.contract('USDC') as ERC20;
+    const COMP = await deploymentManager.contract('COMP') as ERC20;
+    const WBTC = await deploymentManager.contract('WBTC') as ERC20;
+    const WETH = await deploymentManager.contract('WETH') as ERC20;
+    const UNI = await deploymentManager.contract('UNI') as ERC20;
+    const LINK = await deploymentManager.contract('LINK') as ERC20;
+
+    // XXX assert supply caps are currently 0
```

Comment:
Not sure it matters as far as this script is concerned, as long as it makes the right proposal